src/main/kotlin/no/nav/syfo/infrastructure/database/repository/SenOppfolgingRepository.kt

```diff
@@ -124,13 +126,18 @@ class SenOppfolgingRepository(private val database: DatabaseInterface) : ISenOpp
             it.executeQuery().toList { getInt("id") }.single()
         }
 
-    private fun Connection.createVurdering(kandidatId: Int, senOppfolgingVurdering: SenOppfolgingVurdering) =
+    private fun Connection.createVurdering(kandidatId: Int, senOppfolgingVurdering: SenOppfolgingVurdering): PSenOppfolgingVurdering =
         prepareStatement(CREATE_VURDERING).use {
             it.setString(1, senOppfolgingVurdering.uuid.toString())
             it.setInt(2, kandidatId)
             it.setObject(3, senOppfolgingVurdering.createdAt)
             it.setString(4, senOppfolgingVurdering.veilederident)
-            it.setString(5, senOppfolgingVurdering.type.name)
+            if (senOppfolgingVurdering.begrunnelse != null) {
```

Comment:
Dette er strengt tatt ikke nødvendig (for VARCHAR-felter fungerer det å sette til null dersom kolonnen er nullable).

Reply:
Da tester jeg å fjerne det 🙌

Reply:
Hm, satt den til `NOT NULL` i [database migreringen](https://github.com/navikt/ismeroppfolging/pull/34/files) 🤔
Skal vi sette tom `String` for alle begrunnelser? Eller kjøre ny migrering som tillater `null` verdi? Vet vi pratet litt om å gjøre dette optional, men synes det høres litt ut som noe som bør være obligatorisk 🤔 Kan ta en kjapp prat om det i morgen kanskje.

Reply:
Tok en kjapp prat med Anders om det. Vi kan migrere tilbake til at vi tillater `null` når vi finner utav det endelig. Foreløpig kan vi populere med `""` der `begrunnelse` er `null` i dto som kommer inn.